model/src/pyrenew/observation/poisson.py

```diff
@@ -72,4 +72,4 @@ def sample(
             fn=dist.Poisson(rate=mu + self.eps),
             obs=obs,
         )
-        return (poisson_sample,)
+        return (SampledValue(poisson_sample),)
```

Comment:
As above, I think we should probably pass `self.t_start` and `self.t_unit` here, even if they're typically going to be `None`.

Reply:
Noted